companies/tasks/dart_sync.py

```
@@ -168,14 +173,16 @@ def sync_all_reports():
+
 @shared_task
 def sync_all_rankings():
     """
     모든 산업의 시가총액 순위 동기화 (주기적 실행용)
     """
     from companies.tasks.rankings import update_all_rankings_task
+
     try:
         update_all_rankings_task.delay()
         logger.info("산업 시가총액 순위 동기화 작업 등록 완료")
     except Exception as e:
```

Ruff (0.14.13)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)